package.json

```diff
@@ -57,6 +57,7 @@
     "typescript": "^4.3.5"
   },
   "dependencies": {
-    "@openzeppelin/contracts": "^4.2.0"
+    "@openzeppelin/contracts": "^4.2.0",
+    "hardhat-ethernal": "^0.3.7"
```

Comment:
Should this change be here?

Reply:
Who put that there? ;)  No, that was not intentional, and I'm not sure how it got changed.